docs/features/data_in_dynamodb.md

```diff
@@ -59,80 +60,24 @@ cumulus-api CLI also includes a backup and restore command. The CLI backup comma
-
-```
-<deployment_name>:
-  PdrsTable:
-    enableAutoScaling: true  
+```bash
+     ./node_modules/.bin/cumulus-api restore backups/<table-name>.json --table <new-table-name>
 ```
 
-### Defaults
-
-By default, `@cumulus/deployment` will configure auto scaling with the following provisioned throughput values:
-
-|              | Read | Write |
-|--------------|------|-------|
-| Min Capacity | 5    | 1     |
-| Max Capacity | 10   | 2     |
-
-Tables are launched with `Min Capacity` provisioned. To determine when to scale up or down, DynamoDB uses a Cloudwatch alarm to trigger autoscaling whenever read / write throughput exceeds some percentage of the current provisioned througput value. The following values are used to determine how and when to scale:
-
-* **`TargetValue`:** A numeric value between 0 and 100 representing a percentage. When request througput exceeds or falls below `TargetValue` percent of the current provisioned value, DynamoDB scales up or down to `TargetValue` of current throughput.
-* **`TargetValue`** defaults to 30, but can also be overriden in `app/config.yml`.
-* **`ScaleInCooldown`** and **`ScaleOutCooldown`:** A numeric value representing the number of seconds DynamoDB Auto Scaling throughput should exhibit exceeding or subceeding the `TargetValue` percent of provisioned throughput before scaling up or down.
-* **`ScaleInCooldown`** and **`ScaleOutCooldown`** default to 0 but can be overriden in `app/config.yml`
-
-**Read more on the AWS Blog:** [How to use AWS CloudFormation to configure auto scaling for Amazon DynamoDB](https://aws.amazon.com/blogs/database/how-to-use-aws-cloudformation-to-configure-auto-scaling-for-amazon-dynamodb-tables-and-indexes/)
-
-### Example
-
-A fully customized `app/config.yml` for DynamoDB auto scaling might look something like the following:
+## DynamoDB Auto Scaling
 
-```
-<deployment_name>:
-  AutoScalingPolicyConfiguration:
-    targetValue: 70
-    scaleInCooldown: 5
-    scaleOutCooldown: 5
-  CollectionsTable:
-    enableAutoScaling: true
-    ReadMinCapacity: 5
-    ReadMaxCapacity: 10
-    WriteMinCapacity: 1
-    WriteMaxCapacity: 2
-  ProvidersTable:
-    ...
-  RulesTable:
-    ...
-  UsersTable:
-    ...
-  GranulesTable:
-    ...
-  PdrsTable:
-    ...
-  ExecutionsTable:
-    ...  
-  FilesTable:
-    enableAutoScaling: true
-    ReadMinCapacity: 5
-    ReadMaxCapacity: 10
-    WriteMinCapacity: 20
-    WriteMaxCapacity: 100
-```
+Cumulus deployed tables from the [data-persistence module](https://github.com/nasa/cumulus/blob/master/tf-modules/data-persistence) are set to [`on-demand'](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/HowItWorks.ReadWriteCapacityMode.html#HowItWorks.OnDemand) mode.
 
-### Important Note!
+### Important Note
```

Comment:
Is this note still relevant since we're not using autoscaling anymore on the DB tables?

Reply:
No, indeed not.   42dd52457